### ISSUE TITLE
Can this objective function be applied to regression problems

### ISSUE THREAD

**Issue by Malili2018** [ryanchankh/mcr2#12]
I have seen that this MCR has better performance than cross entropy, but when I apply a deep learning model to regression questions, I often choose BCEloss for loss function, so can this objective function be applied to regression problems.

Thanks
